src/textual/_ansi_sequences.py

```diff
@@ -299,3 +304,16 @@
     "\x1b[1;8x": (Keys.Escape, Keys.ControlShift8),
     "\x1b[1;8y": (Keys.Escape, Keys.ControlShift9),
 }
+
+
+# Mapping of escape codes to report whether they support a "mode" we requested.
+ANSI_SEQUENCES_MODE_REPORTS: Dict[str, Tuple[Mode, ModeReportParameter]] = {
+    get_mode_report_sequence(mode, parameter): (mode, parameter)
+    for mode, parameter in [
+        (mode, parameter) for parameter in ModeReportParameter for mode in Mode
+    ]
+}
+
+TERMINAL_MODES_ANSI_SEQUENCES: Dict[Mode, dict] = {
```

Comment:
Typing is imprecise here. I'm assuming the second dict could be `dict[str, str]` ?